openslides/config/serializers.py

```diff
@@ -0,0 +1,9 @@
+from openslides.utils.rest_api import serializers, VariableField
+
+
+class ConfigSerializer(serializers.Serializer):
```

Comment:
The Rest api is your part, so your implementation is ok for me (as long as it works :) )

I would recommend you to use one system. If you use the serializer system from django rest framework, you should use it here as well.